### DISCUSSION TITLE
The ghost organization!

### DISCUSSION THREAD

**Discussion by Redbeanw44602** [community/community#44443] [Repositories]
### Select Topic Area

Bug

### Body

This organization was created by another person, I was set as the owner of the organization, and then the organization was deleted by the person who created it, but I still have this organization in my list.

I can't delete it and if I try to access it returns a 404.

Please help me to delete this organization from my account, thanks!

![image](https://user-images.githubusercontent.com/29711228/212605016-f0c3f8b1-b009-4658-a657-472c0ed9a3ea.png)
https://github.com/LL-Package-Hub

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Redbeanw44602 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, have you tried clearing your browser's cache and cookies? This could possibly related to that. 

I've gone ahead and moved this post over to the correct category for you. Good luck!

**Comment by Redbeanw44602** [comment]
It's not caching, it's even visible on the Github App. I'm not sure what section to use, thanks for your help!
